Code/src/main/java/nl/utwente/viskell/ghcj/GhciSession.java

```diff
@@ -60,6 +60,14 @@ public GhciSession() {
 
         queue = new ArrayBlockingQueue<>(1024);
         errors = EvictingQueue.create(LOG_SIZE);
+        switch (pickBackend()) {
+            case Clash: 
+                this.catalog = new HaskellCatalog("/catalog/clash.xml");
+                break;
+            default:
+                this.catalog = new HaskellCatalog("/catalog/haskell.xml");
+                break;
+        }
```

Comment:
I liked the old setup better, of course - I'd like GhciSession to have as little knowledge of its backends as possible. Maybe make `getCatalogPath` static?